src/main/java/seedu/address/logic/LogicManager.java

```diff
@@ -32,6 +33,7 @@ public class LogicManager implements Logic {
     private final Model model;
     private final Storage storage;
     private final AddressBookParser addressBookParser;
+    private Person lastViewedPerson;
```

Comment:
Is there a reason this was not placed within ViewCommand as a static variable? This feels iffy since none of the other "view" / "list" commands keep their internal state within the LogicManager.java. Perhaps you can share the reason for doing this